### PR TITLE
CI-511: Remove the option to tag team members on the Celebrate page

### PR DESCRIPTION
**Pull request by SerhiiBorozenets**
[CI-511](https://cloverpop-internship.atlassian.net/browse/CI-511)

Note: For migration data from CelebrateShoutouts to Shoutouts please run rake task:
`rake migration:celebrate_shoutouts`

[CI-511]: https://cloverpop-internship.atlassian.net/browse/CI-511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CI-521](https://cloverpop-internship.atlassian.net/browse/CI-521)

[CI-521]: https://cloverpop-internship.atlassian.net/browse/CI-521?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/workers/migrate_shoutouts_worker.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class MigrateShoutoutsWorker
```
Comment (rogergraves) [inline app/workers/migrate_shoutouts_worker.rb:1]: This should only be run once, correct? Maybe put in a TODO to have it removed later (and also add a story for this)

**Review by rogergraves — APPROVED** [review]

File: app/workers/migrate_shoutouts_worker.rb (hunk 1 above)
Comment (SerhiiBorozenets) [reply app/workers/migrate_shoutouts_worker.rb:1]: @martafedori @alosikolesik please create story for this @ @ @

File: app/javascript/components/Pages/CausesToCelebrate.js
Comment (Maryna-Harasko) [inline app/javascript/components/Pages/CausesToCelebrate.js:1]: @SerhiiBorozenets please check Jest test for this file.

**Review by Maryna-Harasko — APPROVED** [review]

**Review by SerhiiBorozenets — COMMENTED** [review]
@LyubovP could you explain why you are removing `app/workers/migrate_shoutouts_worker.rb` and `lib/tasks/migrate_celebrate_shoutouts.rake` in this branch? These needed for migration data from CelebrateShoutout to Shoutout. After this PR is merged with the main one, you should create a separate brunch `CI-521` and make the necessary changes.
@SerhiiBorozenets  could you please see story task  521

**Pull request merged**